virtual-programs/tags-to-quads.folk

```diff
@@ -571,6 +589,42 @@ When camera /camera/ has width /cameraWidth/ height /cameraHeight/ &\
         dict set ::prevTagPoses $tag timestamp $timestamp
         dict set ::prevTagPoses $tag pose $tagPose
 
+        ## tag stabilization ##
+        # (can't use When here as it can't exfiltrate info). Is there a better way to do this?
```

Comment:
by 'exfiltrate' you mean you can't get the information back to the scope outside the When?

you could, I guess, collect all the matches for `/someone/ wishes tag $tag is stabilized` and then do the rest of this body inside there (since that body will always run, whether the result count is 1, 0, or something else). I don't know how that will perform / if it is better but that's probably the right reactive way to do it